tests/test_fltconn_send.py

```diff
@@ -0,0 +1,17 @@
+from modules.communications.FltConnSend import build_fijo_bytearray
+
+
+def test_fltConnEncoding():
+    command = {
+        "type": "QR_COORDINATES",
+        "data": {
+            "longitude": 1.23,
+            "latitude": 1.23,
+            "qr_scan_flag": 1,
+            "takeoff_command": 0,
+            "detect_flag": 1
+        }
+    }
+
+    print(build_fijo_bytearray(command))
```

Comment:
We can update this test later to have a useful assertion, right now it just tests that the code doesn't crash